### ISSUE TITLE
Graphical interface to change settings and rules

### ISSUE THREAD

**Issue by johnste** [johnste/finicky#85]
I'm considering adding more configuration options in the UI of Finicky to allow more users to use it, since not everyone is comfortable with JavaScript.

This would be a completely separate configuration from the current javascript configuration file, and I have yet to determine exactly how/if they would work together.

Work in progress screenshot.
![image](https://user-images.githubusercontent.com/886051/68593983-e31aa080-0496-11ea-9d33-9a3e3684edbf.png)

**Comment by tbrek** [comment]
This looks really good @johnste !

**Comment by rkrueger11** [comment]
@johnste Would love to help anyway I could!

**Comment by johnste** [comment]
Any help here is welcome. I have limited experience working on macOs application interfaces so it's unlikely I would be able to deliver this feature.

I'm looking into alternative methods of making configuring simpler.

**Comment by dillonjohnbrown** [comment]
I don't think this is exactly the kind of "help wanted" you are looking for, but I made a mockup of how I imagine this feature working.

Thank you for making Finicky. I think it's a stellar app, GUI or not.

![Mockup 1](https://dillonbrown.me/images/mockups/finicky-1.png)
![Mockup 2](https://dillonbrown.me/images/mockups/finicky-2.png)

**Comment by johnste** [comment]
Wow, this looks really cool @dillonjohnbrown! Thank you!

**Comment by johnste** [comment]
It's very unlikely that I will be adding features to manage rules directly in the user interface since I'm not very comfortable with that part of MacOs development.
